src/benchmark/CMakeLists.txt

```diff
@@ -44,9 +44,16 @@ target_link_libraries(
 )
 
 # General purpose benchmark runner
-add_executable(hyriseBenchmark benchmark_runner_main.cpp)
+add_executable(
+    hyriseBenchmarkFiles
```

Comment:
That's not a good name. When I hear "hyriseBenchmarkFiles" I think of files describing a benchmark or something. Not of a benchmark executable that can load its data from disk. "hyriseBenchmark" was fine, imo. "hyriseBenchmarkRunner" would be okay with me as well.

Reply:
I'm somewhat trying to get away from the idea that the file-based runner is the "default" one (the way it was implemented before).

Reply:
I agree with you that there shouldn't be "primary" and "secondary" benchmarks. But "hyriseBenchmarkFiles" conveys a different meaning, to me at least, it sounds like "files that make up a benchmark". 
If "hyriseBenchmark" sounds too generic to you, why not "hyriseBenchmarkRunner"? From that name I, at least, could tell that it is an executable that **runs** a generic benchmark that I have specified. What's the generic way to specify a benchmark? Files with queries and files with tables.

Reply:
Hmm, but hyriseBenchmarkTPCH is a benchmarkRunner too, right?
How about "hyriseBenchmarkFileBased", similar to the actual file name?

Reply:
If "FileBased" is really a term we want to use, then, yeah, even that would be better than "hyriseBenchmarkFiles"